### ISSUE TITLE
fix room orders

### ISSUE THREAD

**Issue by pguth66** [pguth66/screeps-pguth66#52]
getting an error where it says rooms have no orders during the market run

**Comment by pguth66** [comment]
Max of 20 orders per player now, apparently, although I'm only at 15.

**Comment by pguth66** [comment]
console output from markethandler run

```[10:15:07 AM][shard1]W27N24 has no orders!
[10:15:07 AM][shard1]W27N27 skipping market for X
[10:15:07 AM][shard1]W28N23 has no orders!
[10:15:07 AM][shard1]W28N24 skipping market for O
[10:15:07 AM][shard1]W28N25 skipping market for L
[10:15:07 AM][shard1]W28N26 has no orders!
[10:15:07 AM][shard1]W28N27 skipping market for X
[10:15:07 AM][shard1]W28N28 has no orders!
[10:15:07 AM][shard1]W29N23 skipping market for L
[10:15:07 AM][shard1]W29N24 has no orders!
[10:15:07 AM][shard1]W29N25 has no orders!
[10:15:07 AM][shard1]W29N26 has no orders!
[10:15:07 AM][shard1]W29N27 has no orders!
[10:15:07 AM][shard1]W29N28 has no orders!
[10:15:07 AM][shard1]W29N29 has no orders!
[10:15:07 AM][shard1]W31N24 skipping market for L
[10:15:07 AM][shard1]W31N25 skipping market for O
[10:15:07 AM][shard1]W32N24 has no orders!
[10:15:07 AM][shard1]W32N25 has no orders!
[10:15:07 AM][shard1]W33N24 skipping market for O
[10:15:07 AM][shard1]W33N25 skipping market for O```

**Comment by pguth66** [comment]
apparently fixed in 873043b90a4bafa4940a033976c653352fb008ed although I'm not sure I get why

**Comment by pguth66** [comment]
more fixed in 3ac4b6010606888339cd0010bef4868809016df1
